src/os_regex/os_regex_strbreak.c

```diff
@@ -43,11 +45,29 @@ char **OS_StrBreak(char match, const char *str, size_t size)
         ret[i] = NULL;
         i++;
     }
+
     i = 0;
 
     while (*str != '\0') {
         i++;
+
+        /* If before match value exists backslash, skip it. */
+        if((count < size - 1) && (*str == match) &&
```

Comment:
Please include the possibility of escaping the separator character via parameter, something like:
`char **OS_StrBreak(char match, const char *str, size_t size, bool scape_match)`
We should add the use of the strcspn() function instead of searching character by character for the separator.